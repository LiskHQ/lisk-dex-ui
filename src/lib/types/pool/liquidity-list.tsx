export interface LiuqidityListInterface {
    token1Id: string;
    token2Id: string;
    feeTier: string;
    pooledtoken1: string;
    pooledtoken2: string;
    poolTokens: string;
    poolShare: string;
    accumulatedFeesToken1: string;
    accumulatedFeesToken2: string;
    totalFees: string;
  }