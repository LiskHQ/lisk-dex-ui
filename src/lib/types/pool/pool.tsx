export interface LiquidityListInterface {
  token1Id: string;
  token2Id: string;
  feeTier: string;
  pooledtoken1: string;
  pooledtoken2: string;
  poolTokens: string;
  poolShare: string;
  accumulatedFeesToken1: string;
  accumulatedFeesToken2: string;
  totalFees: string;
}

export interface IpriceRange {
  minPrice: string,
  maxPrice: string,
  currentPrice: string  
}

export  interface IdepositAmounts {
  token1: string,
  token2: string,
  amount1: string,
  amount2: string,
  feeTier: string,
}
