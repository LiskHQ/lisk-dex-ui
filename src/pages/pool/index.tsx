import { NextPage } from 'next';
import { PoolContainer } from 'containers';
import { withLayout } from 'components';

const PoolPage: NextPage = () => {
  return (
    <PoolContainer />
  );
};

export default withLayout(PoolPage);