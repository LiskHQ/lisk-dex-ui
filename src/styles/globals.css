:root {
  font-size: 16px;
}

html,
body,
#__next {
  padding: 0 !important;
  margin: 0;
  height: 100%;
}

a {
  color: inherit;
  text-decoration: none;
}

* {
  box-sizing: border-box;
}

a,
.btn,
button,
span,
p,
i,
input,
select,
textarea,
li,
img,
svg path,
*::after,
*::before,
.transition-3,
h1,
h2,
h3,
h4,
h5,
h6 {
  transition: all 0.3s ease-out 0s;
}

span,
p,
a,
li.MuiMenuItem-root,
label,
label.MuiFormLabel-root,
span.MuiTypography-root,
p.MuiTypography-root,
button.MuiButton-root,
button.MuiButtonBase-root,
.MuiInput-root,
.MuiTableCell-root {
  font-family: 'Roboto', sans-serif;
}

h1,
h2,
h3,
h4,
h5,
h6,
h1.MuiTypography-root,
h2.MuiTypography-root,
h3.MuiTypography-root,
h4.MuiTypography-root,
h5.MuiTypography-root,
h6.MuiTypography-root {
  font-family: 'Roboto', sans-serif;
}