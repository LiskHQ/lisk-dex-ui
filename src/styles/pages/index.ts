import { GovernanceStyle } from "./governance/index.style";

export {
  GovernanceStyle
}