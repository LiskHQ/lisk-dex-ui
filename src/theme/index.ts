export { default as theme } from "./theme";
export { default as variables } from "./variables";
