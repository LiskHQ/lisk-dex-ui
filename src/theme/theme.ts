import {
  createTheme,
  responsiveFontSizes,
  ThemeOptions,
} from "@mui/material/styles";

import variables from "./variables";

const options: ThemeOptions = {
  palette: {
    primary: variables.primary,
    secondary: variables.secondary,
  },
  typography: {
    h1: {
      fontSize: "2.5rem",
      fontWeight: 600,
    },
    h2: {
      fontSize: "2rem",
      fontWeight: 400,
    },
    h3: {
      fontSize: "1.75rem",
      fontWeight: 400,
    },
    h4: {
      fontSize: "1.25rem",
      fontWeight: 400,
    },
    h5: {
      fontSize: "1rem",
      fontWeight: 400,
    },
    h6: {
      fontSize: "0.75rem",
      fontWeight: 400,
    },
  },
  breakpoints: {
    values: {
      xs: 0,
      sm: 576,
      md: 768,
      lg: 1024,
      xl: 1440,
      xxl: 1950,
      xxxl: 2300,
    },
  },
};

const theme = responsiveFontSizes(createTheme(options));
export default theme;
