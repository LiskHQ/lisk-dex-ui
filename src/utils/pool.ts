export const getPoolToken0 = (pool: string) => {
  return pool.split('-')[0];
};

export const getPoolToken1 = (pool: string) => {
  return pool.split('-')[1];
};