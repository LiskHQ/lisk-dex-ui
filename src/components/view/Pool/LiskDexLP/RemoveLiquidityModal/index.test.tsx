import React from 'react';
import { ThemeProvider } from '@mui/material';
import { fireEvent, render } from '@testing-library/react';
import { lightTheme } from 'styles/theme';
import { mockPools } from '__mock__/pool.mock';
import { IRemoveLiquidityModalProps, RemoveLiquidityModal } from '.';

function renderComponent(props: IRemoveLiquidityModalProps) {
  return render(
    <ThemeProvider theme={lightTheme} >
      <RemoveLiquidityModal {...props} />
    </ThemeProvider>
  );
}

describe('LiskDexLP component', () => {
  const mockProps: IRemoveLiquidityModalProps = {
    pool: mockPools[0],
    onPreview: jest.fn(),
    onClose: jest.fn(),
  };

  it('checks if the component matches the b  snapshot', () => {
    const { container } = renderComponent(mockProps);
    expect(container).toMatchSnapshot();
  });

  it('click on preview', () => {
    const { getByTestId } = renderComponent(mockProps);
    fireEvent.click(getByTestId('remove-liquidity-modal-preview-test'));
    expect(mockProps.onPreview).toBeCalled();
  });

  it('click on close', () => {
    const { getByTestId } = renderComponent(mockProps);
    fireEvent.click(getByTestId('remove-liquidity-modal-cancel-test'));
    expect(mockProps.onClose).toBeCalled();
  });
});
