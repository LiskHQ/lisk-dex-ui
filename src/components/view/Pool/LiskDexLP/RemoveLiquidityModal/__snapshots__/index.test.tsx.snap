// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LiskDexLP component checks if the component matches the b  snapshot 1`] = `
.emotion-0 .remove-liquidity-modal-background {
  position: fixed;
  left: 0;
  top: 0;
  width: 100%;
  height: 100%;
  background: rgba(0, 0, 0, 0.5);
  z-index: 30;
}

.emotion-0 .remove-liquidity-modal-container {
  position: fixed;
  left: 50%;
  top: 50%;
  margin-bottom: 10rem;
  -webkit-transform: translate(-50%, -50%);
  -moz-transform: translate(-50%, -50%);
  -ms-transform: translate(-50%, -50%);
  transform: translate(-50%, -50%);
  width: 30.5rem;
  border-radius: 0.5rem;
  background: #FFFFFF;
  box-shadow: 0px 1px 7px rgba(0, 0, 0, 0.1);
  z-index: 30;
  padding: 1.5rem;
}

@media (max-width:575.95px) {
  .emotion-0 .remove-liquidity-modal-container {
    width: 100%;
  }
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-modal-title {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-modal-title .MuiIconButton-root svg path {
  stroke: #282828;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-modal-summmary {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-top: 1.5rem;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-modal-summmary .summary-token-name {
  margin-left: 0.5rem;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-modal-summmary .summary-rate {
  background: #D9D8F8;
  padding: 0.25rem 0.5rem;
  border-radius: 0.5rem;
  margin-left: 1rem;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-modal-summmary .summary-rate p {
  font-size: 0.75rem;
  line-height: 1rem;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-amount {
  margin-top: 2.5rem;
  margin-bottom: 1.5rem;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-amount h1 {
  text-align: center;
  margin-top: 1.25rem;
  margin-bottom: 1.5rem;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-amount .amount-selections {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -ms-flex-pack: space-around;
  -webkit-justify-content: space-around;
  justify-content: space-around;
  margin-top: 1rem;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-amount .amount-selections .MuiButtonBase-root {
  padding: 0.5rem 1rem;
  border-radius: 0.5rem;
  width: 3.75rem;
  height: 2.25rem;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-amount .amount-selections .MuiButtonBase-root.MuiButton-outlined {
  color: #6953F4;
  border: 1px solid #E6E6E6;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-amount .MuiSlider-root .MuiSlider-rail {
  background: #F4F8F9;
  height: 1rem;
  border-radius: 1rem;
  opacity: 1;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-amount .MuiSlider-root .MuiSlider-track {
  height: 1rem;
  background: #6953F4;
  border-radius: 1rem;
  opacity: 1;
  border: none;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-amount .MuiSlider-root .MuiSlider-thumb {
  width: 1.5rem;
  height: 1.5rem;
  border: 2px solid #6953F4;
  background: #FFFFFF;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-will-receive {
  margin-top: 1rem;
  margin-bottom: 1.5rem;
  padding: 1rem;
  background: #FAFAFC;
  border: 1px solid #E6E6E6;
  border-radius: 0.5rem;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-will-receive .token-amount {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-will-receive .token-amount.token2 {
  margin-top: 0.75rem;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-will-receive .token-amount .title {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-0 .remove-liquidity-modal-container .remove-liquidity-will-receive .token-amount .title p {
  margin-left: 0.5rem;
}

.emotion-4 {
  margin: 0;
  font-size: 1.25rem;
  line-height: 1.5rem;
  font-weight: 600;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.00735em;
}

.emotion-5 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  text-align: center;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  font-size: 1.5rem;
  padding: 8px;
  border-radius: 50%;
  overflow: visible;
  color: rgba(0, 0, 0, 0.54);
  -webkit-transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-5::-moz-focus-inner {
  border-style: none;
}

.emotion-5.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-5 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-5:hover {
  background-color: rgba(0, 0, 0, 0.04);
}

@media (hover: none) {
  .emotion-5:hover {
    background-color: transparent;
  }
}

.emotion-5.Mui-disabled {
  background-color: transparent;
  color: rgba(0, 0, 0, 0.26);
}

.emotion-6 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-8 {
  margin: 0;
  font-size: 1rem;
  line-height: 1.375rem;
  font-weight: 500;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.00938em;
}

.emotion-10 {
  margin: 0;
  font-size: 0.875rem;
  line-height: 1.125rem;
  font-weight: 400;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.01071em;
}

.emotion-13 {
  margin: 0;
  font-size: 2.5rem;
  font-weight: 600;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  line-height: 1.167;
  letter-spacing: -0.01562em;
}

.emotion-14 {
  border-radius: 12px;
  box-sizing: content-box;
  display: inline-block;
  position: relative;
  cursor: pointer;
  touch-action: none;
  color: #1976d2;
  -webkit-tap-highlight-color: transparent;
  height: 4px;
  width: 100%;
  padding: 13px 0;
}

@media (pointer: coarse) {
  .emotion-14 {
    padding: 20px 0;
  }
}

@media print {
  .emotion-14 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-14.Mui-disabled {
  pointer-events: none;
  cursor: default;
  color: #bdbdbd;
}

.emotion-14.MuiSlider-dragging .MuiSlider-thumb,
.emotion-14.MuiSlider-dragging .MuiSlider-track {
  -webkit-transition: none;
  transition: none;
}

.emotion-15 {
  display: block;
  position: absolute;
  border-radius: inherit;
  background-color: currentColor;
  opacity: 0.38;
  width: 100%;
  height: inherit;
  top: 50%;
  -webkit-transform: translateY(-50%);
  -moz-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
}

.emotion-16 {
  display: block;
  position: absolute;
  border-radius: inherit;
  border: 1px solid currentColor;
  background-color: currentColor;
  -webkit-transition: left 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,width 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,bottom 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,height 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: left 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,width 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,bottom 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,height 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  height: inherit;
  top: 50%;
  -webkit-transform: translateY(-50%);
  -moz-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
}

.emotion-17 {
  position: absolute;
  width: 20px;
  height: 20px;
  box-sizing: border-box;
  border-radius: 50%;
  outline: 0;
  background-color: currentColor;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-transition: box-shadow 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,left 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,bottom 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: box-shadow 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,left 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,bottom 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  top: 50%;
  -webkit-transform: translate(-50%, -50%);
  -moz-transform: translate(-50%, -50%);
  -ms-transform: translate(-50%, -50%);
  transform: translate(-50%, -50%);
}

.emotion-17:before {
  position: absolute;
  content: "";
  border-radius: inherit;
  width: 100%;
  height: 100%;
  box-shadow: 0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);
}

.emotion-17::after {
  position: absolute;
  content: "";
  border-radius: 50%;
  width: 42px;
  height: 42px;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%, -50%);
  -moz-transform: translate(-50%, -50%);
  -ms-transform: translate(-50%, -50%);
  transform: translate(-50%, -50%);
}

.emotion-17:hover,
.emotion-17.Mui-focusVisible {
  box-shadow: 0px 0px 0px 8px rgba(25, 118, 210, 0.16);
}

@media (hover: none) {
  .emotion-17:hover,
  .emotion-17.Mui-focusVisible {
    box-shadow: none;
  }
}

.emotion-17.Mui-active {
  box-shadow: 0px 0px 0px 14px rgba(25, 118, 210, 0.16);
}

.emotion-17.Mui-disabled:hover {
  box-shadow: none;
}

.emotion-19 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 0.875rem;
  line-height: 1.75;
  letter-spacing: 0.02857em;
  text-transform: uppercase;
  min-width: 64px;
  padding: 5px 15px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border: 1px solid rgba(25, 118, 210, 0.5);
  color: #1976d2;
  width: 100%;
  color: #FF4557;
  padding: 0.75rem;
  text-transform: capitalize;
}

.emotion-19::-moz-focus-inner {
  border-style: none;
}

.emotion-19.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-19 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-19:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: rgba(25, 118, 210, 0.04);
  border: 1px solid #1976d2;
}

@media (hover: none) {
  .emotion-19:hover {
    background-color: transparent;
  }
}

.emotion-19.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
  border: 1px solid rgba(0, 0, 0, 0.12);
}

.emotion-19.MuiButton-outlined {
  border-color: #FF4557;
}

.emotion-19.MuiButton-contained {
  background: #6953F4;
  color: #FFFFFF;
}

.emotion-19.Mui-disabled {
  opacity: 0.7;
}

.emotion-22 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 0.875rem;
  line-height: 1.75;
  letter-spacing: 0.02857em;
  text-transform: uppercase;
  min-width: 64px;
  padding: 6px 16px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  color: #fff;
  background-color: #1976d2;
  box-shadow: 0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);
  width: 100%;
  color: #FF4557;
  padding: 0.75rem;
  text-transform: capitalize;
}

.emotion-22::-moz-focus-inner {
  border-style: none;
}

.emotion-22.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-22 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-22:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: #1565c0;
  box-shadow: 0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);
}

@media (hover: none) {
  .emotion-22:hover {
    background-color: #1976d2;
  }
}

.emotion-22:active {
  box-shadow: 0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12);
}

.emotion-22.Mui-focusVisible {
  box-shadow: 0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12);
}

.emotion-22.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
  box-shadow: none;
  background-color: rgba(0, 0, 0, 0.12);
}

.emotion-22.MuiButton-outlined {
  border-color: #FF4557;
}

.emotion-22.MuiButton-contained {
  background: #6953F4;
  color: #FFFFFF;
}

.emotion-22.Mui-disabled {
  opacity: 0.7;
}

<div>
  <div
    class="emotion-0"
    data-testid="remove-liquidity-modal-test"
  >
    <div
      class="remove-liquidity-modal-background MuiBox-root emotion-1"
    />
    <div
      class="remove-liquidity-modal-container MuiBox-root emotion-1"
    >
      <div
        class="remove-liquidity-modal-title MuiBox-root emotion-1"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4 emotion-4"
        >
          Increase liquidity
        </h4>
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-5"
          data-testid="remove-liquidity-modal-cancel-test"
          tabindex="0"
          type="button"
        >
          <svg
            fill="none"
            height="20"
            viewBox="0 0 20 20"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              clip-path="url(#clip0_625_48983)"
            >
              <path
                d="M15 5L5 15"
                stroke="#282828"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="1.66667"
              />
              <path
                d="M5 5L15 15"
                stroke="#282828"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="1.66667"
              />
            </g>
            <defs>
              <clippath
                id="clip0_625_48983"
              >
                <rect
                  fill="white"
                  height="20"
                  width="20"
                />
              </clippath>
            </defs>
          </svg>
          <span
            class="MuiTouchRipple-root emotion-6"
          />
        </button>
      </div>
      <div
        class="remove-liquidity-modal-summmary MuiBox-root emotion-1"
      >
        <img
          alt="DEX"
          height="24"
          src="https://raw.githubusercontent.com/LiskHQ/app-registry/main/devnet/Dex/images/tokens/dex.png"
          style="border-radius: 100%;"
          width="24"
        />
        <img
          alt="ENVT"
          height="24"
          src="https://raw.githubusercontent.com/LiskHQ/app-registry/main/devnet/Enevti/images/tokens/enevti.png"
          style="border-radius: 100%;"
          width="24"
        />
        <p
          class="MuiTypography-root MuiTypography-body1 summary-token-name emotion-8"
        >
          DEX
           / 
          DEX
        </p>
        <div
          class="summary-rate MuiBox-root emotion-1"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 emotion-10"
          >
            0.05%
          </p>
        </div>
      </div>
      <div
        class="remove-liquidity-amount MuiBox-root emotion-1"
      >
        <p
          class="MuiTypography-root MuiTypography-body2 amount-title emotion-10"
        >
          Amount
        </p>
        <h1
          class="MuiTypography-root MuiTypography-h1 emotion-13"
        >
          50
          %
        </h1>
        <span
          class="MuiSlider-root MuiSlider-colorPrimary MuiSlider-sizeMedium emotion-14"
        >
          <span
            class="MuiSlider-rail emotion-15"
          />
          <span
            class="MuiSlider-track emotion-16"
            style="left: 0%; width: 50%;"
          />
          <span
            class="MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary MuiSlider-thumb MuiSlider-thumbSizeMedium MuiSlider-thumbColorPrimary emotion-17"
            data-index="0"
            style="left: 50%;"
          >
            <input
              aria-orientation="horizontal"
              aria-valuemax="100"
              aria-valuemin="0"
              aria-valuenow="50"
              data-index="0"
              max="100"
              min="0"
              step="1"
              style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
              type="range"
              value="50"
            />
          </span>
        </span>
        <div
          class="amount-selections MuiBox-root emotion-1"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium emotion-19"
            tabindex="0"
            type="button"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 emotion-8"
            >
              25
              %
            </p>
            <span
              class="MuiTouchRipple-root emotion-6"
            />
          </button>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-22"
            tabindex="0"
            type="button"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 emotion-8"
            >
              50
              %
            </p>
            <span
              class="MuiTouchRipple-root emotion-6"
            />
          </button>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium emotion-19"
            tabindex="0"
            type="button"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 emotion-8"
            >
              75
              %
            </p>
            <span
              class="MuiTouchRipple-root emotion-6"
            />
          </button>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium emotion-19"
            tabindex="0"
            type="button"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 emotion-8"
            >
              +
            </p>
            <span
              class="MuiTouchRipple-root emotion-6"
            />
          </button>
        </div>
      </div>
      <p
        class="MuiTypography-root MuiTypography-body2 emotion-10"
      >
        You will receive
      </p>
      <div
        class="remove-liquidity-will-receive MuiBox-root emotion-1"
      >
        <div
          class="token-amount MuiBox-root emotion-1"
        >
          <div
            class="token1 title MuiBox-root emotion-1"
          >
            <img
              alt="DEX"
              height="24"
              src="https://raw.githubusercontent.com/LiskHQ/app-registry/main/devnet/Dex/images/tokens/dex.png"
              style="border-radius: 100%;"
              width="24"
            />
            <p
              class="MuiTypography-root MuiTypography-body1 emotion-8"
            >
              DEX
            </p>
          </div>
          <p
            class="MuiTypography-root MuiTypography-body1 emotion-8"
          >
            1862
          </p>
        </div>
        <div
          class="token-amount token2 MuiBox-root emotion-1"
        >
          <div
            class="title token2 MuiBox-root emotion-1"
          >
            <img
              alt="ENVT"
              height="24"
              src="https://raw.githubusercontent.com/LiskHQ/app-registry/main/devnet/Enevti/images/tokens/enevti.png"
              style="border-radius: 100%;"
              width="24"
            />
            <p
              class="MuiTypography-root MuiTypography-body1 emotion-8"
            >
              ENVT
            </p>
          </div>
          <p
            class="MuiTypography-root MuiTypography-body1 emotion-8"
          >
            0.045
          </p>
        </div>
      </div>
      <p
        class="MuiTypography-root MuiTypography-body2 emotion-10"
      >
        Prices
      </p>
      <div
        class="remove-liquidity-will-receive MuiBox-root emotion-1"
      >
        <div
          class="token-amount MuiBox-root emotion-1"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 emotion-8"
          >
            1 
            DEX
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 emotion-8"
          >
            0.00345 
            ENVT
          </p>
        </div>
        <div
          class="token-amount token2 MuiBox-root emotion-1"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 emotion-8"
          >
            1 
            ENVT
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 emotion-8"
          >
            2138.42 
            DEX
          </p>
        </div>
      </div>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium remove-liquidity-modal-confirm emotion-22"
        data-testid="remove-liquidity-modal-preview-test"
        tabindex="0"
        type="button"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 emotion-8"
        >
          Preview
        </p>
        <span
          class="MuiTouchRipple-root emotion-6"
        />
      </button>
    </div>
  </div>
</div>
`;
